.github/actions/gh-api-update-status/action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
   gh-token:
     description: "The GitHub token for use with the CLI"
     required: true
+  repository:
```

Comment:
This is needed if we ever want to test this on a fork (like I did, during development of this fix)